### PR TITLE
ua reduction changes

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes b/300473118

Changes proposed in this pull request:

- https://pr-7360-static-dot-dcc-staging.uc.r.appspot.com/blog/force-major-version-to-100/
- https://pr-7360-static-dot-dcc-staging.uc.r.appspot.com/blog/user-agent-reduction-android-model-and-version/
- https://pr-7360-static-dot-dcc-staging.uc.r.appspot.com/blog/user-agent-reduction-deprecation-trial/
- https://pr-7360-static-dot-dcc-staging.uc.r.appspot.com/blog/user-agent-reduction-oct-2022-updates/
- https://pr-7360-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/user-agent/
- https://pr-7360-static-dot-dcc-staging.uc.r.appspot.com/blog/user-agent-reduction-origin-trial/
- partial: site/en/_partials/privacy-sandbox/timeline/ua-ch.md
-

### CODE REVIEW

File: site/en/blog/user-agent-reduction-deprecation-trial/index.md [hunk 1]
```diff
@@ -1,22 +1,32 @@
 ---
 layout: "layouts/blog-post.njk"
-title: "User-Agent Reduction deprecation trial"
+title: "User-Agent Reduction deprecation trial has now ended"
```
Comment (anusmitaray) [inline site/en/blog/user-agent-reduction-deprecation-trial/index.md:3]: Is our plan to update all blogs after the fact? I thought blogs were very much a snapshot of the moment, and the assumption was that they would get outdated with time?

File: site/en/docs/privacy-sandbox/user-agent/index.md [hunk 2]
```diff
@@ -2,39 +2,39 @@
...
-updated: 2023-08-01
+updated: 2023-09-23
 authors:
   - alexandrawhite
 ---
 
-## Implementation status
+<!-- ## Implementation status
+
+The following list indicates previous stages in the gradual User-Agent string reduction. The reduction is now complete and the origin and deprecation trials are closed.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/user-agent/index.md:16]: Is this meant to be hidden?

File: site/en/blog/user-agent-reduction-deprecation-trial/index.md (hunk 1 above)
Comment (nancymic2) [reply site/en/blog/user-agent-reduction-deprecation-trial/index.md:3]: I had that exact question. It's up to you all.

File: site/en/docs/privacy-sandbox/user-agent/index.md (hunk 2 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/user-agent/index.md:16]: I wanted to indicate that I think it should go

**Review by anusmitaray — APPROVED** [review]

File: site/en/blog/user-agent-reduction-deprecation-trial/index.md (hunk 1 above)
Comment (nancymic2) [reply site/en/blog/user-agent-reduction-deprecation-trial/index.md]: reverted text changes. added aside about being outdated.

File: site/en/docs/privacy-sandbox/user-agent/index.md (hunk 2 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/user-agent/index.md]: removed status

**Pull request merged**